app/models/user.rb

```diff
@@ -7,4 +7,16 @@ class User < ApplicationRecord
   PRE_RELEASE_FEATURES_PERMISSION = "pre_release_features"
   DEBUG_PERMISSION = "debug"
   MANAGING_EDITOR_PERMISSION = "managing_editor"
+  GDS_ACCESS_LIMIT_OVERRIDE_PERMISSION = "gds_access_limit_override"
+
+  def can_access?(edition)
+    return true unless edition.access_limit
+
+    return true if has_permission?(
+      User::GDS_ACCESS_LIMIT_OVERRIDE_PERMISSION,
```

Comment:
minor, but you shouldn't need the `User::` qualifier here.

Reply:
👍